back-end/aws-cloud9-api/src/main/java/com/serverless/dal/Form.java

```diff
@@ -1,128 +1,103 @@
-            .build();
-        // get the db adapter
-        this.db_adapter = DynamoDBAdapter.getInstance();
-        this.client = this.db_adapter.getDbClient();
-        // create the mapper with config
-        this.mapper = this.db_adapter.createDbMapper(mapperConfig);
-    }
-
-  
-    @Override
-	public String toString() {
-		return "Form [getId()=" + getId() + ", getName()=" + getName() + ", getDescription()=" + getDescription() + "]";
+	
+	 private String id;
+	 private String name;
+	 private String creationDate;
+	 private List<Map<String,String>> users;
+	 //private Map<String,String> users[];
+	 
+	 
+	 public Form() {
+		super();
+		this.id = null;
+		this.name = null;
+		this.creationDate = null;
+		this.users = new ArrayList<Map<String,String>>();
+		//this.users = null;
 	}
-	// methods
-    public Boolean ifTableExists() {
-        return this.client.describeTable(FORMS_TABLE_NAME).getTable().getTableStatus().equals("ACTIVE");
-    }
-
-    public List<Form> list() throws IOException {
-    	DynamoDBScanExpression scanExp = new DynamoDBScanExpression();
-        List<Form> results = this.mapper.scan(Form.class, scanExp);
-        return results;
-    }
-    
-    public Form get(String id) throws IOException {
-    	Form form = null;
-
-        HashMap<String, AttributeValue> av = new HashMap<String, AttributeValue>();
-        av.put(":v1", new AttributeValue().withS(id));
-
-        DynamoDBQueryExpression<Form> queryExp = new DynamoDBQueryExpression<Form>()
-            .withKeyConditionExpression("id = :v1")
-            .withExpressionAttributeValues(av);
-
-        PaginatedQueryList<Form> result = this.mapper.query(Form.class, queryExp);
-        if(result.size() > 0) {
-        	form = result.get(0);
-        }
-        return form;
-    }
-    
-    public void save() throws IOException {
-        this.mapper.save(this);
-    }
-    
-    public Boolean delete(String id) throws IOException {
-        Form form = null;
+	 	@DynamoDBHashKey(attributeName = "id")
+	    @DynamoDBAutoGeneratedKey
+	    public String getId() {
+	        return this.id;
+	    }
+	    public void setId(String id) {
+	        this.id = id;
+	    }
+
+	    @DynamoDBRangeKey(attributeName = "name")
+	    public String getName() {
+	        return this.name;
+	    }
+	    
+	    public void setName(String name) {
+	        this.name = name;
+	    }
+	    
+	    @DynamoDBAttribute(attributeName = "creationDate")
+	    public String getCreationDate() {
+	        return this.creationDate;
+	    }
+	    
+	    public void setCreationDate(String creationDate) {
+	        this.creationDate = creationDate;
+	    }
+	    
+	    @DynamoDBAttribute(attributeName = "users")
+	    public void addUsers(String userID, String recruiterID) {
+	    	Map<String, String> map = new HashMap<String,String>();
+	    	map.put("userID",userID);
+	    	map.put("recruitedID", recruiterID);
+	    	this.users.add(map);
+	    }
```

Comment:
To jest do zmiany w związku ze zmianą typu zmiennej `users`

Reply:
Tak jak wytłumaczyłem wcześniej, nie zmieniałem ze względu na Mateusza